diff --git a/src/map/ai/states/attack_state.cpp b/src/map/ai/states/attack_state.cpp
index fd7d52f97e..2f4938d55f 100644
--- a/src/map/ai/states/attack_state.cpp
+++ b/src/map/ai/states/attack_state.cpp
@@ -134,8 +134,8 @@ void CAttackState::UpdateTarget(const EntityId& target)
             {
                 for (auto&& PPotentialTarget : PChar->SpawnMOBList)
                 {
-                    if (PPotentialTarget.second->animation == xi::Animation::Attack && facing(PChar->loc.p, PPotentialTarget.second->loc.p, 64) &&
-                        distance(PChar->loc.p, PPotentialTarget.second->loc.p) <= 10)
+                    if (PPotentialTarget.second->animation == xi::Animation::Attack &&
+                        distance(PChar->loc.p, PPotentialTarget.second->loc.p) <= 20.0f)
                     {
                         std::unique_ptr<CBasicPacket> errMsg;
                         if (PChar->IsValidTarget(EntityId(PPotentialTarget.second), TARGET_ENEMY, errMsg))
