diff --git a/scripts/combat/physical_utilities.lua b/scripts/combat/physical_utilities.lua
index 126aacdc55..402569fe05 100644
--- a/scripts/combat/physical_utilities.lua
+++ b/scripts/combat/physical_utilities.lua
@@ -908,7 +908,11 @@ xi.combat.physical.calculateRangedPDIF = function(actor, target, weaponType, wsA
         pDif = pDif * 1.25
     end
 
-    -- Step 5: TODO: True Shot.
+    -- Step 5: True Shot.
+    local trueShotBonus = actor:getMod(xi.mod.TRUE_SHOT_EFFECT)
+    if trueShotBonus > 0 then
+        pDif = pDif * (1 + trueShotBonus / 100)
+    end
 
     -- Crit damage bonus is a final modifier
     if isCritical then
